lib/Applify.pm

```diff
@@ -350,6 +350,16 @@ sub _print_synopsis {
   unless (-e $documentation) {
     eval "use $documentation; 1" or die "Could not load $documentation: $@";
     $documentation =~ s!::!/!g;
+    # might be FatPacked $INC{"$documentation.pm"}{"$documentation.pm"}
+    if (ref($INC{"$documentation.pm"})) {
```

Comment:
How about `if (ref $INC{"$documentation.pm"} eq 'HASH')` ?

Reply:
I know a lot more about `require` now. The `%INC` entry is a `FatPacked` object that must have an `INC` method which returns a readable filehandle.